packages/docs/components/Recaptcha.vue

```diff
@@ -0,0 +1,49 @@
+<template>
```

Comment:
Would recommend a 2 word name for this component.

Reply:
Would suggest leaving it like this, I respect the guidelines with clear component naming but Recaptcha is clear enough 🙂